src/lib.rs

```diff
@@ -161,6 +161,16 @@ pub extern "C" fn GA_get_networks(ret: *mut *const GA_json) -> i32 {
 #[cfg(feature = "android_logger")]
 static INIT_LOGGER: Once = ONCE_INIT;
 
+#[no_mangle]
+pub extern "C" fn GA_init(_config: *const GA_json) -> i32 {
+    debug!("GA_init()");
```

Comment:
Maybe log the config object passed in, for debugging purposes?

Reply:
Good idea. Done.